demo/index.html

```diff
@@ -19,13 +19,13 @@
 <body>
 <div class="form-container">
     <h6>Upload CSV Files</h6>
-    <div id="cdeFileDropzone" class="dropzone">
-        <input type="file" id="cdeFileInput" hidden>
-        <p>Click to select a Dictionary file</p>
+    <div id="datasetMappingFileDropzone" class="dropzone">
```

Comment:
Renames dictionary to dataset mapping for consistency with the nomenclature used in the application.